common/unsafe/pom.xml

```diff
@@ -47,6 +47,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.scala-lang.modules</groupId>
+      <artifactId>scala-parallel-collections_${scala.binary.version}</artifactId>
+      <version>1.0.4</version>
```

Comment:
1. Do you need to bind to this particular version?
2. How about to limit the scope to `test`.